Sources/Plasma/FeatureLib/pfMoviePlayer/plMoviePlayer.cpp

```diff
@@ -335,7 +335,7 @@ void plMoviePlayer::IProcessVideoFrame(const std::vector<blkbuf_t>& frames)
     }
     
     // We need to refill the buffer if there is a new image or a new buffer
-    if (img || !fTexture->GetDeviceRef()) {
+    if (img || (!fTexture->GetDeviceRef() && fLastImg)) {
```

Comment:
The crash on Windows was here. Windows may enter the draw loop before a frame is decompressed (not sure why Mac doesn't.) That means we need to make sure fLastImage isn't null before we try to use it.

@Hoikas - give this a try. I'm away from my home setup for the next few weeks, so I'm just getting my Windows laptop stood up. I'm not set up to get all the way into an age, but the intro movie does now complete successfully.